apps/wallet-mobile/src/features/Staking/Governance/useCases/Home/HomeScreen.tsx

```diff
@@ -0,0 +1,390 @@
+import {isNonNullable, isString} from '@yoroi/common'
+import {
+  GovernanceProvider,
+  useDelegationCertificate,
+  useGovernance,
+  useLatestGovernanceAction,
+  useStakingKeyState,
+  useVotingCertificate,
+} from '@yoroi/staking'
+import React, {ReactNode, useEffect, useState} from 'react'
+import {StyleSheet, Text, View} from 'react-native'
+
+import {Button, Spacer, useModal} from '../../../../../components'
+import {useStakingInfo} from '../../../../../Dashboard/StakePoolInfos'
+import {useUnsafeParams, useWalletNavigation} from '../../../../../navigation'
+import {useSelectedWallet} from '../../../../../SelectedWallet'
+import {useStakingKey, useTransactionInfos, useWalletEvent} from '../../../../../yoroi-wallets/hooks'
+import {TransactionInfo} from '../../../../../yoroi-wallets/types'
+import {Action, BrokenImage, LearnMoreLink, useNavigateTo, useStrings} from '../../common'
+import {mapStakingKeyStateToGovernanceAction} from '../../common/helpers'
+import {Routes} from '../../common/navigation'
+import {GovernanceVote} from '../../types'
+import {EnterDrepIdModal} from '../EnterDrepIdModal'
+
+export const HomeScreen = () => {
+  const wallet = useSelectedWallet()
+  const txInfos = useTransactionInfos(wallet)
+  const stakingKeyHash = useStakingKey(wallet)
+  const [isPendingRefetchAfterTxConfirmation, setIsPendingRefetchAfterTxConfirmation] = useState(false)
+
+  const {data: stakingStatus, refetch: refetchStakingKeyState} = useStakingKeyState(stakingKeyHash, {
+    refetchOnMount: true,
+    suspense: true,
+  })
+
+  useWalletEvent(wallet, 'utxos', refetchStakingKeyState)
+
+  const {data: lastSubmittedTx} = useLatestGovernanceAction(wallet.id)
+
+  const submittedTxId = lastSubmittedTx?.txID
+
+  const isTxPending = isString(submittedTxId) && !isTxConfirmed(submittedTxId, txInfos)
+
+  useEffect(() => {
```

Comment:
import * as React...yadayada.
```suggestion
  React.useEffect(() => {
```